email/src/main/java/edu/jhuapl/dorset/demos/EmailTypes.java

```diff
@@ -0,0 +1,40 @@
+package edu.jhuapl.dorset.demos;
+
+public enum EmailTypes {
```

Comment:
Let's make this singular as it gets used like a name type. You should be able to right click on the class and select refactor to change the name to EmailType.